package.json

```diff
@@ -19,10 +16,6 @@
     "ripple-lib": "0.13.0-rc6.0",
     "simple-jsonrpc": "~0.0.2"
   },
-  "scripts": {
-    "pretest": "node test/pretest.js",
-    "test": "mocha test/websocket-test.js test/server-test.js test/*-test.{js,coffee}"
-  },
```

Comment:
I *think* this file is only used in the context of `npm`, so it can be deleted, too.